variables.tf

```diff
@@ -72,4 +72,4 @@ variable "encrypt_ebs" {
 variable "volume_type" {
   type    = string
   default = "gp3"
```

Comment:
Do we know if there is a new volume type better than this `gp3`, like a `gp4`?

Reply:
No idea. But, we can do that in another MR.